### ISSUE TITLE
Autosave comments (& forms)

### ISSUE THREAD

**Issue by philippejadin** [agorakit/agorakit#31]
garder 30 minutes puis effacer, sauver en ajax

**Comment by petrk94** [comment]
does the function work now?

**Comment by philippejadin** [comment]
No, I considered the issue was stale (trying to focus the todo list), but your comments tells me that it's not :-).

Now let's find a good way to make it happen

**Comment by philippejadin** [comment]
I'd tend to use unpolly form observers and detect in code that it's an autosave instead of full save.
or something client side like that : https://github.com/depuits/jQuery.AutoSaveForm#readme

Is it something you need a lot ?

**Comment by linc** [comment]
I think the concept of "drafts" and autosaving is neat, but I'd honestly move this to a ways down the roadmap. It adds a bunch of complexity (I've found it increases bugs & edge cases around posting) and it doesn't have a big impact on whether folks decide to use it.
